packages/app/src/app/sync/sync.service.ts

```diff
@@ -42,15 +45,25 @@ export class SyncService {
   private _connections = new BehaviorSubject<Connection[]>([]);
 
   constructor() {
-    // Reload the map every 60s if nothing changed
-    const noChanges$ = this.observeConnections()
-      .pipe(
-        filter(con => con.length > 0),
-        switchMap(() => this._state.observeMapState()),
-        debounceTime(60_000)
-      )
-
-    merge(this._session.observeOperationId(), this._session.observeIsOnline(), this._session.observeLabel(), noChanges$)
+    // Reload the websocket every 15min if nothing changed
+    // each _reconnect try(respectively _disconnect) will set _connections again to [] and emit again
+    const noChanges$ = this.observeConnections().pipe(
+      filter((con) => con.length > 0),
+      switchMap(() => this._state.observeMapState()),
+      debounceTime(RECONNECT_ACTIVE_CONNECTION_TIME),
```

Comment:
Now it reconnects the websocket every 9 seconds. I guess the value of the RECONNECT_ACTIVE_CONNECTION_TIME is wrong or to low for every 15 minutes it should be 900_000?